runtime/src/lib.rs

```diff
@@ -275,10 +278,36 @@ parameter_types! {
     pub MinSolutionScoreBump: Perbill = Perbill::from_rational_approximation(5u32, 10_000);
 }
 
+/// Struct that handles the conversion of Balance -> `u64`. This is used for staking's election
+/// calculation.
+pub struct CurrencyToVoteHandler;
+
+impl CurrencyToVoteHandler {
+    fn factor() -> Balance {
+        (orml_tokens::MultiTokenCurrencyAdapter::<Runtime>::total_issuance(NATIVE_CURRENCY_ID)
```

Comment:
it seems like it will evaluate to `0` each time when total_issuance is < u64::max_value() ? is that expected behaviour ?

Reply:
there's .max(1) to prevent that